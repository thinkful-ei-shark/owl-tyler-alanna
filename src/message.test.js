import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Message from './message';


it ('renders without crashing', ()=> {
    const div = document.createElement('div');

    ReactDOM.render(<Message type= 'clap' name = 'test' timestamp = {1548852544247}/>, div);

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Message type= 'clap' name = 'test' timestamp = {1548852544247}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });