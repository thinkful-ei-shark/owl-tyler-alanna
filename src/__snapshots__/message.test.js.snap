// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="event"
>
  <h3>
    test
  </h3>
  <p>
    clapped!
  </p>
  <p
    className="timestamp"
  >
    1548852544247
  </p>
</div>
`;
